tests/testStreamOpen.cpp

```diff
@@ -460,3 +460,69 @@ TEST_F(StreamOpenOutput, OboeExtensions){
         ASSERT_TRUE(OboeExtensions::isMMapEnabled());
     }
 }
+
+TEST_F(StreamOpenInput, AAudioInputSetPrivacySensitiveModeUnspecifiedMedia){
+    if (getSdkVersion() >= __ANDROID_API_R__){
```

Comment:
You could run these tests on <R and then conditionally choose different expected results. Then we can check if it causes crashes.

Reply:
Added a separate test for <R below. Adding more if statements makes the test confusing